### DISCUSSION TITLE
create-razzle-app cache folder .gitignore

### DISCUSSION THREAD

**Discussion by stygianlgdonic** [jaredpalmer/razzle#1669] [Q&A] [answered]
Hi, Is it okay to add cache folder to .gitignore?
Reason why i'm asking this:
- whenever i global search the folder for some keyword it shows results from cache folder as well which i do not want to see if possible.
- without any git changes (clean tree, nothing staged/unstaged changes) if i run npm start, there are changes in cache folder.

So is it okay to ignore this folder?

**Comment by fivethreeo** [comment, marked as answer]
Ok to ignore
